src/uniprotkb/components/entry/CommunityCuration.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  ExternalLink,
+  HeroContainer,
+  CommunityAnnotationIcon,
+  ChevronDownIcon,
+  Card,
+  InfoList,
+} from 'franklin-sites';
+import cn from 'classnames';
+
+import useDatabaseInfoMaps from '../../../shared/hooks/useDatabaseInfoMaps';
+
+import { processUrlTemplate } from '../../../shared/utils/xrefs';
+import externalUrls from '../../../shared/config/externalUrls';
+
+import { Reference } from '../../../supporting-data/citations/adapters/citationsConverter';
+import EntrySection from '../../types/entrySection';
+
+import styles from './styles/community-curation.module.scss';
+
+import ORCIDiDLogo from '../../../images/ORCIDiD_icon.png';
+
+const CommunityCuration = ({
+  accession,
+  section,
+  communityReferences,
+}: {
+  accession: string;
+  section: EntrySection;
+  communityReferences: (Reference | undefined)[];
+}) => {
+  const databaseInfoMaps = useDatabaseInfoMaps();
+
+  if (communityReferences.length) {
+    return (
+      <details className={styles['community-annotation-details']}>
```

Comment:
I couldn't see the arrow rotate like I can see in other places we use `<details` eg BLAST form Advanced parameters. Is that behaviour intentional?

Reply:
There was a typo in the class name. The arrow at the end flips as expected now